src/Flex.php

```diff
@@ -198,7 +227,7 @@ public function executeAutoScripts(Event $event): void
         $this->io->write($this->postInstallOutput);
     }
 
-    private function filterPackageNames(PackageInterface $package, string $operation)
+    private function filterPackageNames(PackageInterface $package, string $operation): ?\Generator
```

Comment:
As soon as the body of a fonction contains a yield, it will return a generator (possibly an empty one): https://3v4l.org/FgHWh